app/build.gradle

```diff
@@ -122,7 +122,6 @@ dependencies {
     implementation 'androidx.constraintlayout:constraintlayout:1.1.3'
     implementation "androidx.browser:browser:$androidx_version"
```

Comment:
Nice find! This also seems to be unused: do you mind removing it and double checking nothing looks weird at runtime?

Reply:
It used in `SafeIntentKtTest`